rdkit/ML/UnitTestAnalyzeComposite.py

```diff
@@ -52,7 +52,7 @@ def test1_Issue163(self):
 
     self.assertEqual(res[0][0],'BALABANJ')
     self.assertEqual(res[1][0],'BERTZCT')
-    self.assertEqual(res[-1][0],'FR_UNBRCH_ALKANE')
+    self.assertEqual(res[-1][0],'VSA_ESTATE9')
```

Comment:
Python 3.6 now preserves the order of dictionaries. Therefore, the original code failed (FR_ALLYLIC_OXID). Greg then changed sort order of the results in the `AnalyzeComposite.ProcessIt` and at that point FR_UNBRCH_ALKANE was the last one. Now we sort in the test code and VSA_ESTATE9 is last (as it should be).